firmware/raspi/boardinterface.py

```diff
@@ -229,15 +233,24 @@ def send_message_to_arduino(self, board_move):
 
         print(f"Sending message \"{msg}\" to arduino")
 
-        # TODO: Implement sending message to arduino
+        
```

Comment:
`ser.write(msg)` can be directly after the `print` statement (don't need a comment or blank lines)